apps/api-extractor/src/collector/Collector.ts

```diff
@@ -292,9 +292,9 @@ export class Collector {
   private _makeUniqueNames(): void {
     const usedNames: Set<string> = new Set<string>();
 
-    // First collect the explicit package exports
+    // First collect the explicit package exports (named)
     for (const entity of this._entities) {
-      if (entity.exported) {
+      if (entity.exported && entity.originalName !== 'default') {
```

Comment:
> 'default' [](start = 52, length = 10)

Better to use `ts.InternalSymbolName.ExportStar` instead of handcoding this string.

Reply:
Fixed

---
In reply to: [246239738](https://github.com/Microsoft/web-build-tools/pull/1008#discussion_r246239738) [](ancestors = 246239738)